gk/main.c

```diff
@@ -653,6 +727,9 @@ cleanup_gk(struct gk_config *gk_conf)
 {
 	int i;
 
+	if (ipv6_if_configured(&gk_conf->net->front))
+		destroy_ipv6_acls(&gk_conf->net->front);
```

Comment:
GK didn't include any ACL in the IPv6 filter, so why should it clear anything here? The same goes for the GT block.